src/PersonRow.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from './types';
+import classNames from 'classnames';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonRow: React.FC<Props> = ({ person }) => {
+  const { name, sex, born, died, fatherName, motherName, mother, father } =
+    person;
+
+  const { personId } = useParams();
+  const FEMALE = 'f';
+  const MINUS = '-';
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': personId === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames({
+            'has-text-danger': FEMALE === sex,
+          })}
+        >
+          {name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {mother ? (
+          <Link to={`/people/${mother.slug}`} className="has-text-danger">
+            {mother.name}
+          </Link>
```

Comment:
create the <PersonLink person={person} /> and use it for all existing people;
![image](https://github.com/mate-academy/react_people-table-basics/assets/63243136/5607c5d5-af90-46f8-9409-a2c52e18c768)